cassovary-core/src/main/scala/com/twitter/cassovary/util/io/FileReader.scala

```diff
@@ -85,4 +85,24 @@ class TwoTsFileReader[T](fileName: String,
     (source, dest)
   }
 
+}
+
+// T is the id type, typically Int or Long or String
+class AdjacencyTsFileReader[T](fileName: String,
```

Comment:
Instead of having "Any", you should change it (T, Option[Int]) -- where T is either the source or dest id and "Int" represents the count of adjacency.

Also please add a comment about this.